src/SFA.DAS.LevyTransferMatching.Web/Validators/Opportunities/ContactDetailsPostRequestValidator.cs

```diff
@@ -0,0 +1,53 @@
+﻿using FluentValidation;
+using SFA.DAS.LevyTransferMatching.Web.Models.Opportunities;
+using System.Net.Mail;
+
+namespace SFA.DAS.LevyTransferMatching.Web.Validators.Opportunities
+{
+    public class ContactDetailsPostRequestValidator : AbstractValidator<ContactDetailsPostRequest>
```

Comment:
I can't recall if email address have to be unique. Maybe we weren't asked to enforce that - I'll leave it with you.

Reply:
Checked with Sam, and it appears they have to be unique. Checking with #ltm-main, the error message in this case should be "You have already entered this email address". This has been implemented now.